server/src/graphql/resolvers/queries.js

```diff
@@ -1,22 +1,19 @@
 const bcrypt = require('bcryptjs');
+
 const { db } = require('../../db/models');
 const { isValidEmail } = require('../../utils');
+const errorMessages = require('../../constants/errors');
 
 module.exports = {
   login: async (_, { model: { email, password } }) => {
-    const isValid = isValidEmail(email);
-    const invalidUser = {
-      isValid: false,
-    };
-    if (!isValid) {
-      return invalidUser;
-    }
+    const emailIsValid = isValidEmail(email);
+    if (!emailIsValid) return new Error(errorMessages.invalidEmail);
```

Comment:
`if` statements without `{}` surrounding the logic tend to be harder to read than leaving out the `{}`